jupyterhub/roles.py

```diff
@@ -214,7 +232,7 @@ def _existing_only(func):
     """Decorator for checking if roles exist"""
 
     @wraps(func)
-    def _check_existence(db, entity, role=None, *, rolename=None):
+    def _check_existence(db, entity, role=None, commit=True, *, rolename=None):
```

Comment:
can this be keyword-only (after the `*`)?

Reply:
Done in 0d6778f95584f3e5fc200125bf185c2809a60358.